databricks/koalas/plot.py

```diff
@@ -946,8 +946,32 @@ def kde(self, bw_method=None, ind=None, **kwds):
     def pie(self, bw_method=None, ind=None, **kwds):
         return _unsupported_function(class_name='pd.DataFrame', method_name='pie')()
 
-    def area(self, bw_method=None, ind=None, **kwds):
-        return _unsupported_function(class_name='pd.DataFrame', method_name='area')()
+    def area(self, x=None, y=None, **kwds):
+        """
+        Draw a stacked area plot.
+
+        An area plot displays quantitative data visually.
+        This function wraps the matplotlib area function.
+
+        Parameters
+        ----------
+        x : label or position, optional
+            Coordinates for the X axis. By default uses the index.
+        y : label or position, optional
+            Column to plot. By default uses all columns.
+        stacked : bool, default True
+            Area plots are stacked by default. Set to False to create a
+            unstacked plot.
+        **kwds : optional
+            Additional keyword arguments are documented in
+            :meth:`DataFrame.plot`.
+
+        Returns
+        -------
+        matplotlib.axes.Axes or numpy.ndarray
+            Area plot, or array of area plots if subplots is True.
```

Comment:
@charlesdong1991, after you finish to add all plots, mind cleaning those documentation to have some examples in a separate PR later? It would be great if we do it in Seres's too together.

Reply:
yeah, honestly I was thinking of doing so :) once most of plots are finished, I will create a separate PR to fix them all for Series and DataFrame. I will focus on plot implementation for now.

Does it sound good for you? @HyukjinKwon

Reply:
Yea, plase!